frontend/src/app/App.tsx

```diff
@@ -31,6 +32,10 @@ export const App = (): JSX.Element => (
     <BrowserRouter>
       <ChakraProvider theme={theme} resetCSS>
         <AuthProvider>
+          <Box height="2rem" width="100vw" position="fixed" top="0" zIndex="11">
+            <GovtMasthead />
+          </Box>
+          <Box height="2rem" width="100vw" />
```

Comment:
Should we move this into a component for cleanliness, say `AppMasthead`? May also want to explore using a `Portal` instead of using `zIndex`, see

https://chakra-ui.com/docs/components/recipes/z-index

Reply:
thanks! moved to `AppMasthead` as suggested. Also as discussed, investigated using `Portal` but will just stick to `zIndex` for now